Src/Support/Google.Apis.Auth.Tests/OAuth2/Responses/TokenResponseTests.cs

```diff
@@ -44,57 +51,182 @@ public void TestSerializer()
             var tokenResponse = JsonConvert.DeserializeObject<TokenResponse>(
                 @"{
                  'access_token': '123',
+                 'id_token': '321',
                  'expires_in': 1000,
                  'refresh_token': '456',
                  'scope': '789'
                 }");
             Assert.Equal("123", tokenResponse.AccessToken);
+            Assert.Equal("321", tokenResponse.IdToken);
             Assert.Equal("456", tokenResponse.RefreshToken);
             Assert.Equal("789", tokenResponse.Scope);
             Assert.Equal(1000, tokenResponse.ExpiresInSeconds);
         }
 
-        [Fact]
-        public void IsExpired()
+        public static IEnumerable<object[]> ExpiresData
```

Comment:
A comment would be handy for this.

Reply:
(In particular, explaining the use of MemberData rather than InlineTheoryData would be helpful. I believe I understand now, but I had to work it out.)